microsoft-edge/devtools-guide-chromium/network-request-blocking/network-request-blocking-tool.md

```diff
@@ -21,42 +21,47 @@ After you create blocked network requests and test the webpage, you can then edi
+1. In DevTools, on the Activity Bar, click the **Network request blocking** tab.  If that tab isn't visible, click the **More Tools** (![More Tools icon](./network-request-blocking-tool-images/more-tools-icon-light-theme.png)) button.
 
 1. Click the **Add pattern** (![Add pattern icon](./network-request-blocking-tool-images/add-pattern-icon.png)) button.  The **Enable network request blocking** checkbox is automatically selected.
 
-1. In the **Text pattern to block network requests** text box, type the URL of a network request that you want to block.  You can either type the full URL, just the domain name to block all requests from this domain, or replace parts of it with `*` for wildcard pattern matching.
+1. In the **Text pattern to block matching requests** text box, type **\*.jpg** to block all requests for JPEG images.
+
+   You can either type the full URL, just the domain name to block all requests from this domain, or replace parts of it with `*` for wildcard pattern matching.
    
    For example, `contoso.com` matches URLs like:
 
    * `https://contoso.com`
    * `https://subdomain.contoso.com`
    * `https://subdomain.contoso.com/path/to/resource`
 
-   And `*.png` matches URLs like:
+   And `*.jpg` matches URLs like:
    
-   * `https://www.contoso.com/resource.png`
-   * `http://third-party.com/6469272/163348534-b90ea1a3-c33cbeb1aed8.png`
+   * `https://www.contoso.com/resource.jpg`
+   * `http://third-party.com/6469272/163348534-b90ea1a3-c33cbeb1aed8.jpg`
 
 1. Click the **Add** button:
 
-   ![Blocking the https://*.contoso.com/* URL pattern in the network request blocking tool](./network-request-blocking-tool-images/block-network-request.png)
+   ![The Network request blocking tool, showing the new *.jpg blocking pattern](./network-request-blocking-tool-images/block-network-request.png)
+
+1. Refresh the page.  Now all JPEG images are blocked, the number of blocked network requests is indicated in the **Network request blocking** tool, and the images in the rendered webpage are missing:
+
+   ![The Network request blocking tool, blocking all JPEG images](./network-request-blocking-tool-images/blocked-images.png)
 
 
 <!-- ====================================================================== -->
 ## Delete a blocked network request
 
-To delete a specific network blocking request:
+To delete a specific network blocking request pattern:
 
-*  In the **Network request blocking** table, hover over the network blocking request, and then click the **Remove** (![Remove blocked request icon](./network-request-blocking-tool-images/remove-blocked-request-icon.png)) button:
+*  In the **Network request blocking** table, hover over the network blocking request pattern, and then click the **Remove** (![Remove blocked request icon](./network-request-blocking-tool-images/remove-blocked-request-icon.png)) button:
```

Comment:
```suggestion
*  In the **Network request blocking** table, hover over a network blocking request pattern, and then click the **Remove** (![Remove blocked request icon](./network-request-blocking-tool-images/remove-blocked-request-icon.png)) button:
```
consider "a", to hint that there are multiple of such items, rather than seeming to refer to a single UI item.